Sources/ArcGISToolkit/Components/Compass/Compass.swift

```diff
@@ -125,4 +132,13 @@ public extension Compass {
         copy.size = size
         return copy
     }
+    
+    /// Specifies whether the ``Compass`` should automatically hide when the heading is 0.
+    /// - Parameter newAutomaticallyHides: A Boolean value indicating whether  the compass should automatically
+    /// hide/show itself when the heading is `0`.
+    func automaticallyHides(_ newAutomaticallyHides: Bool) -> some View {
```

Comment:
```suggestion
    func automaticallyHides(_ flag: Bool) -> some View {
```

After some thought I think `automaticallyHides` is a good name, how about the parameter's name though?

Apple uses [`flag`](https://developer.apple.com/documentation/familycontrols/familyactivitypicker/allowstightening(_:)#:~:text=Parameters-,flag,value%20that%20indicates%20whether%20the%20space%20between%20characters%20compresses%20when%20necessary.,-Return%20Value) for a similar parameter to a modifier.